Source/WebCore/page/csp/ContentSecurityPolicySourceList.h

```diff
@@ -60,6 +60,9 @@ class ContentSecurityPolicySourceList {
     bool allowUnsafeHashes() const { return m_allowUnsafeHashes; }
     bool shouldReportSample() const { return m_reportSample; }
 
+    HashAlgorithmSet reportHash() const { return m_reportHash; }
+
+
```

Comment:
```suggestion
    HashAlgorithmSet reportHash() const { return m_reportHash; }

```

Reply:
I see this was unclear. I was suggesting one blank line rather than than two.

Reply:
done